src/main/kotlin/org/wfanet/measurement/eventdataprovider/privacybudgetmanagement/Composition.kt

```diff
@@ -62,8 +70,8 @@ object Composition {
       var deltaI = 0.0f
       for (l in 0..i - 1) {
         deltaI +=
-          coeff(k, l).toFloat() *
-            (exp(epsilon * (k - l).toFloat()) - exp(epsilon * (k - 2 * i + l).toFloat()))
+          exp(logBinomial(k, l) + (epsilon * (k - l).toFloat())) -
+            exp(logBinomial(k, l) + epsilon * (k - 2 * i + l).toFloat())
       }
       deltaI /= (1.0f + exp(epsilon)).pow(k)
```

Comment:
Wondering if there's a benefit to also take log of denominator and put it inside exp?